doc/modules/feature_selection.rst

```diff
@@ -234,11 +234,27 @@ Randomized sparse models
 
 .. currentmodule:: sklearn.linear_model
 
-The limitation of L1-based sparse models is that faced with a group of
-very correlated features, they will select only one. To mitigate this
-problem, it is possible to use randomization techniques, reestimating the
-sparse model many times perturbing the design matrix or sub-sampling data
-and counting how many times a given regressor is selected.
+In terms of feature selection, there are some well-known limitations of
+L1-penalized models for regression and classification.  For example, it is
+known that the Lasso will tend to select an individual variable out of a group
+of highly correlated features.  Furthermore, even when the correlation between
+features is not too high, the conditions under which L1-penalized methods
+consistently select "good" features can be restrictive in general.
+
+To mitigate this problem, it is possible to use a randomization technique
+known as stability selection.  In this method, a subsample of the data is fit
+to a L1-penalized model where the penalty of a random subset of coefficients
+has been scaled.  Specifically, given a subsample of the data :math:`(y_i,
```

Comment:
It is the penalty of a random subset of coefficients the thing that is scaled, and not the coefficients themselves.  It is in the paper right above theorem 2. In the paper's notation, W_k are iid bernoulli with Pr(W_k=\alpha)=Pr(W_k=1)=1/2. So, effectively, the penalty of a random subset is scaled by 1/\alpha. In this implementation, the particular thing is the selection of Pr(W_k=\alpha)=Pr(W_k=1)=1/2.